### DISCUSSION TITLE
useStore equivalent in nanostores/lit

### DISCUSSION THREAD

**Discussion by lschierer** [ai/nanostores#243] [Q&A]
The [best practices section](https://github.com/nanostores/nanostores#best-practices) states that I should "[r]educe get() usage outside of tests" should use useStore instead.  As that is not available in the lit integration package, what would be the corresponding behavior?  The mixin example on the [nanostores/lit](https://github.com/nanostores/lit) page uses the .get() call, and I'm not sure it is avoidable?

**Comment by ai** [comment]
If you already subscribed to the store, it is OK to call `get()`.

But current API may force users to forget subscribing. Maybe we need to create an issue in that repo (sorry, I am not Lit user to help).
